src/server/internalServer.h

```diff
@@ -44,13 +44,20 @@ extern "C" {
 namespace kiwix {
 
 struct GeoQuery {
+  GeoQuery()
+    : GeoQuery(0, 0, -1)
+  {}
+
   GeoQuery(float latitude, float longitude, float distance)
     : latitude(latitude), longitude(longitude), distance(distance)
   {}
   float latitude;
   float longitude;
   float distance;
 
+  operator bool() const {
```

Comment:
We don't have that risk now, but for safety such conversion must be declared `explicit`.

Reply:
Done